src/RealtimeChannel.ts

```diff
@@ -198,6 +199,7 @@ export default class RealtimeChannel {
 
     this.broadcastEndpointURL =
       httpEndpointURL(this.socket.endPoint) + '/api/broadcast'
+    this.private = this.params.config.private || false
```

Comment:
recommend to make this one more secure by default:

```suggestion
    this.private = this.params.config.private !== false
```

Reply:
every improvement is good! 🔥

Reply:
won't undefined or null default to true? 🤔

Reply:
yea exactly just making sure that the only time the channel is not private is when the user explicitly sets `false`. if they accidentally pass in `undefined` or `null` then channel is private.

this could also prevent hammering the database b/c if the private flag is true then it will hit the database to check policies.

Reply:
now that i think about it though in edge case scenarios we might want to do the opposite so prevents users from accidentally setting private to true and hammering the database b/c private true with hit database to check policies.

anyways, i'm good with whatever you decide.

Reply:
this will actually break current clients as they will start to send private messages but not be handling them properly. if we do it we should switch that flag at a later stage and release it as a major with a big depreciation warning telling users that the default behaviour is private: true

Reply:
going to revert this change before merge to avoid breaking current channel implementations and will move this default to true in a client major release so people are better informed about it